source/proof-of-concept-guide/poc-vulnerability-detection.rst

```diff
@@ -3,25 +3,31 @@
 .. meta::
    :description: Wazuh detects if installed applications have an unpatched CVE in the monitored system. Learn more about this in this PoC.
 
-Vulnerability detection
+Vulnerability Detection
 =======================
 
-Wazuh uses the Vulnerability Detection module to identify vulnerabilities in applications and operating systems running on endpoints.
+The Wazuh Vulnerability Detection module plays a crucial role in identifying vulnerabilities in applications and operating systems installed on endpoints. This module actively scans and reports on unpatched Common Vulnerabilities and Exposures (CVEs), enhancing the security posture of the monitored environments.
 
-This use case shows how Wazuh detects unpatched Common Vulnerabilities and Exposures (CVEs) in the monitored endpoint.
-
-For more information on this capability, check the :doc:`vulnerability detection </user-manual/capabilities/vulnerability-detection/index>` section of the documentation.
+For a comprehensive understanding of this feature, refer to the :doc:`Vulnerability Detection section </user-manual/capabilities/vulnerability-detection/index>` in the Wazuh documentation.
 
 Infrastructure
 --------------
 
-+---------------+--------------------------------------------------------------------------------------------------------------------------------------------+
-| Endpoint      | Description                                                                                                                                |
-+===============+============================================================================================================================================+
-| Ubuntu 22.04  | The vulnerability detection module checks for vulnerabilities in the operating system and installed applications in this Linux endpoint.   |
-+---------------+--------------------------------------------------------------------------------------------------------------------------------------------+
-| Windows 11    | The vulnerability detection module checks for vulnerabilities in the operating system and installed applications in this Windows endpoint. |
-+---------------+--------------------------------------------------------------------------------------------------------------------------------------------+
+The infrastructure setup for this PoC includes a Wazuh Manager hosted on Ubuntu 22.04 and multiple Agents deployed across various operating systems. These Agents continuously monitor and evaluate the security state of their respective systems.
```

Comment:
Let's better keep the same format across all PoCs in this guide. "Infrastructure" sub-section uses just a table and no introductory text. The table lists the endpoints used in the use case. This listed endpoints correspond to the alerts showed in the "Visualize the alerts" section. These are very specific endpoints. The ones particularly used to later display alerts. For example:

![imagen](https://github.com/wazuh/wazuh-documentation/assets/47069802/8ecc541e-0285-418c-9c7d-a57fcc2a03b9)